presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/ConfigAPI.java

```diff
@@ -44,7 +42,6 @@
 public final class ConfigAPI
 {
     private static final APIFileHandler apiFileHandler = new APIFileHandler(Paths.get("/etc/presto"));
-    private static final Logger LOGGER = LogManager.getLogger(ConfigAPI.class);
```

Comment:
All HTTP requests are logged by Airlift itself.
though I've noticed there is a bit less detail... We might want to add these back to all of the API classes later.
Better yet, we might be able to use Jersey's `LoggingFeature` to do it for us.